docs/src/Box.doc.js

```diff
@@ -494,6 +502,48 @@ card(
   />
 );
 
+card(
+  <Card
+    description={`
+    Borders are controlled by the "borderSize" prop. Setting a size enables a solid light gray color in that width.
+  `}
+    id="border"
+    name="Borders"
+  >
+    <Box display="flex" wrap>
+      {['sm', 'lg'].map((size, i) => (
+        <Box
+          column={4}
+          mdColumn={3}
+          lgColumn={2}
+          key={i}
+          padding={4}
+          display="flex"
+          direction="column"
+          alignItems="center"
+        >
+          <Box marginBottom={2}>
+            <Text align="center" size="sm">
+              borderSize=&quot;{size}&quot;
+            </Text>
+          </Box>
+          <Box position="relative" padding={4}>
+            <Box position="relative">
+              <Box
+                color="gray"
```

Comment:
@jennyscript Should we make this color `white` so it's easier to see the difference between 1 and 2px? (since the border color is lightgray)